source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/skin.html

```
@@ -253,7 +245,8 @@
 		     (see body:has(#sidenavContent .credits) in the CSS). -->
 		<div class='caCreditsActions'>
 			<div class='caButton caCreditsStatistics' role='button' tabindex='0'><?tr("Statistics");?></div>
-			<div class='caButton caCreditsChangeLog' role='button' tabindex='0'><?tr("Change Log");?></div>
+			<div class='caButton caCreditsChangeLog' role='button' tabindex='0'><?tr("Changes");?></div>
+			<div class='caButton caCreditsDebugging' role='button' tabindex='0'><span class='caCreditsLogsLabel'><?tr("Logs");?></span><?if (($GLOBALS['caSettings']['dev'] ?? "no") == "yes"):?><span class='caCreditsLogsHint'><i class='fa fa-keyboard-o'></i>&nbsp;<i class='fa fa-arrow-up'></i>&nbsp;D</span><?endif;?></div>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)